tests/test_isrFunctions.py

```diff
@@ -254,7 +254,7 @@ def test_illumCorrection(self):
         with self.assertRaises(RuntimeError):
             ipIsr.illuminationCorrection(self.mi, flatMi, 1.0, trimToFit=False)
 
-    def test_overscanCorrection_isInt(self):
+    def notest_overscanCorrection_isInt(self):
```

Comment:
Why have these been disabled? There's no note in the commit message, and it's not clear what "to check for Jenkins level failures" means.
Besides disabling the tests, there are other functional changes, which should probably be part of a different commit.

Reply:
This has been removed, as the coverage is handled by the `test_overscanCorrection.py` tests.